app/helpers/allergens_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AllergensHelper
```

Comment:
I don't think that we will need a helper for this, but we will drop it later if needed.